I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/room0.rpy", line 637, in script
    eat_en ah "With this floor plan, you can know which rooms are accessible and which are not. It also shows which rooms we have already been through."
Exception: Sayer 'eat_en' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/room0.rpy", line 637, in script
    eat_en ah "With this floor plan, you can know which rooms are accessible and which are not. It also shows which rooms we have already been through."
  File "D:\renpy-8.0.3-sdk\renpy-8.0.3-sdk\renpy\ast.py", line 890, in execute
    who = eval_who(self.who, self.who_fast)
  File "D:\renpy-8.0.3-sdk\renpy-8.0.3-sdk\renpy\ast.py", line 783, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'eat_en' is not defined.

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.1.23060707
MeMary 1.0
Mon Dec 25 21:16:38 2023
